src/main/webapp/app/core/auth/passkey-authentication-page/passkey-authentication-page.component.html

```diff
@@ -3,7 +3,37 @@
         <div class="col-md-10 col-lg-8">
             <div class="card shadow">
                 <div class="card-body p-4 p-md-5">
-                    <jhi-passkey-content (triggerPasskeyLoginSuccessHandler)="redirectToOriginalUrlOrHome()" />
+                    <div class="text-center mb-4">
+                        <h3>
+                            <fa-icon [icon]="faLock" />
+                            <span jhiTranslate="global.menu.admin.loginWithPasskeyToUseAdminFeatures"></span>
+                        </h3>
```

Comment:
Was previously in `passkey-prompt.component`, which was a wrapper to prevent code duplication when we needed the modal.

We do not need the modal anymore, so we can also get rid of the wrapper component to simplify the code